### ISSUE TITLE
Documentation contributions

### ISSUE THREAD

**Issue by oameye** [JuliaDynamics/CriticalTransitions.jl#83]
https://github.com/Ferrite-FEM/Ferrite.jl/blob/master/CONTRIBUTING.md#documentation

**Comment by reykboerner** [comment]
Is any action needed on this issue @oameye ?

**Comment by oameye** [comment]
yeah I thought it would be cool to have this, once we have some users :)
